basics/answers.md

```diff
@@ -156,7 +156,15 @@ isn't exactly like mine but it works just fine it's ok, and you can
     print(message, "!!!")
     print(message, "!!!")
     ```
-
+3. Here, we create  a new variable `palindrome_check`.We use this variable to store the  reversed  string(of`palindrome_input`). Then we check if the string that was given(`palindrome_input`)
+   and the reversed sting(`palindrome_check`) is the same using a simple `if loop`
```

Comment:
I thought I added a review comment about this last time, but apparently I forgot:
- This explanation hasn't gotten outdated as the code below has changed.
- It is not an if loop.
- We should mention what `[::-1]` does, even though the reader is expected to have googled it.

```suggestion
3. In the code below, `palindrome_input[::-1]` is the string `palindrome_input` reversed.
   For example, if `palindrome_input` is `"hello"`, then `palindrome_input[::-1]` is `"olleh"`.
```